README.md

```diff
@@ -26,9 +26,16 @@ The DICOM 3D standard is an alternative solution to sharing of 3D ultrasound dat
 This approach has advantages over the traditional DICOM data standard technique including complete seamless backward compatibility to existing stored studies, smaller files sizes, faster network transfers, and faster loading of studies (since no conversion from vendor raw data to DICOM format is required). This new approach also allows the vendors to continually innovate as well, instead of being locked into a frozen data specification.
 
 ## Getting started
-Install [Visual Studio](https://visualstudio.microsoft.com/) 2015 or 2017, either professional or community edition. Enable the ".Net desktop development", "Desktop development with C++" and "Python development" workloads during installation.
-In addition, Python.exe need to be registered as default app for running .py scripts. This can be done by right-clicking on a .py file, then selecting "Open with", "Choose another app", locate Python.exe and enable "Always use this app to open .py files". 
 
+### Tools 
+* Install [Visual Studio](https://visualstudio.microsoft.com/) 2015 or 2017, either professional or community edition. Enable the following workloads during installation: 
+    * ".Net desktop development"
+    * "Desktop development with C++" 
+    * and "Python development" .
+    * Also make sure to tick "Windows Universal CRT SDK" and all versions of "Windows 10 SDK" 
```

Comment:
Thanks for helping to improve the documentation.

Some comments:
* Isn't the "Windows Universal CRT SDK" automatically installed when selecting "Desktop development with C++"?
* Can you please explain why all versions of "Windows 10 SDK" are needed? Isn't it sufficient to just install the default version installed when selecting "Desktop development with C++"?

Reply:
> Isn't the "Windows Universal CRT SDK" automatically installed when selecting "Desktop development with C++"?

Indeed I had actually explicitly selected "Desktop development with C++". Nevertheless it "Windows Universal CRT SDK" was missing for me and the latest Git HEAD version and build stopped working after pulling latest version

> Can you please explain why all versions of "Windows 10 SDK" are needed? Isn't it sufficient to just install the default version installed when selecting "Desktop development with C++"?

In theory yes, ultimately I ended up even manually changing it ("Retarget"). So prb this part of my change should be revised - then again I don't think we should have to change anything manually in the build files. Actually would it even make sense to think of using CMake instead? That might actually avoid this type of issues (but I guess it would also have high impact on all the integration that might be around by now). At least in my case I have to use it with VS2013, where I downgraded build files (which then again would prb lock out the integration with manifest file)

Reply:
Please note that VS2013 is not supported, so you're on your own if using such an old version of Visual Studio.

I'll try to reproduce your findings in a Win10 VM tomorrow and get back to you afterwards.

Reply:
> Please note that VS2013 is not supported, so you're on your own if using such an old version of Visual Studio.

Sure, no worries - it is working fine for me anyways (both GE and Philips readers) :-)

> I'll try to reproduce your findings in a Win10 VM tomorrow and get back to you afterwards.

👍

Reply:
just to clarify: for what I reported above I was using VS2017. In any case pls feel free to adapt or reject, depending on what you find out - maybe I really missed sth during install

Reply:
and finally: it's not actually urgent for me, I am already able to move forward with my current integration of the previous version. I simply wanted to report that I ran into trouble when pushing those ITK export scripts #159

Reply:
I just tried to install VS2017 in a clean Win10 VM with the following workloads enabled:
![image](https://user-images.githubusercontent.com/2671400/77296725-7c30b200-6ce8-11ea-98ef-e8ad74f9c7dc.png)

The Image3dAPI projects built fine without any problems afterwards:
![image](https://user-images.githubusercontent.com/2671400/77296853-b26e3180-6ce8-11ea-91ca-26ad98b67c6b.png)

Reply:
same settings here atually, just minor version update from the installer missing (15.9.20 instead of 15.9.21), which may have had a bug? Anyways thansks for testing!

![image](https://user-images.githubusercontent.com/52462518/77315333-ac3b7d80-6d07-11ea-941d-f6a003ca8a68.png)